modules/tinymce/src/core/main/ts/keyboard/DeleteBackspaceKeys.ts

```diff
@@ -100,7 +100,11 @@ const executeKeyupOverride = (editor: Editor, evt: KeyboardEvent, isBackspaceKey
       ...isBackspaceKeydown ? [{
         // Firefox detects macOS Command key code as "Command" not "Meta"
         keyCode: isFirefox ? 224 : 91,
-        action: MatchKeys.action(InlineFormatDelete.refreshCaret, editor)
+        action: MatchKeys.action(() => {
+          InlineFormatDelete.updateCaretFormat(editor, formatNodes);
+          InlineFormatDelete.refreshCaret(editor);
+          return true;
```

Comment:
Even though `refreshCaret` returns true today, that might not always be the case. The return value of the `action` function determines whether following actions are attempted.
```suggestion
          return InlineFormatDelete.refreshCaret(editor);
```